plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-expected-output-lib.yml

```diff
@@ -21,145 +21,117 @@ project:
     - id: "SPM:apple:swift-crypto:2.6.0"
     - id: "SPM:apple:swift-log:1.5.3"
     - id: "SPM:apple:swift-metrics:2.4.1"
-    - id: "SPM:apple:swift-nio:2.61.0"
+    - id: "SPM:apple:swift-nio:2.58.0"
```

Comment:
This is a downgrade? Let's see if this passes...

Reply:
Unbelievable how flaky it is today. Inbetween NuGetFunTest failed, which is now green again, but the SPM dep has been upgraded again.

Reply:
latest test run again has another result for SPM, which again different than my local result.

@sschuberth shall we just merge this as-is ? (and fix SPM on top later on)

Reply:
Yes, let's fix one thing at a time today... I'll merge.